docs/homebrew/nx-shell.md

```diff
@@ -1,9 +1,8 @@
-### **NX-Shell**
-NX-Shell is a file manager.
+# NX-Shell
 
------
+NX-Shell is a file manager.
```

Comment:
Unrelated to this PR, but probably should expand this to 'NX-Shell is an on-console file manager, able to be launched from the homebrew menu'. Such a short introduction sounds odd to me.